python/pacemaker/_cts/audits.py

```diff
@@ -264,30 +264,40 @@ def __init__(self, cm):
         self.known = []
         self.name = "FileAudit"
 
+    def _output_has_core(self, output, node):
+        found = False
+
+        for line in output:
+            line = line.strip()
+
+            if line in self.known:
+                continue
+
+            found = True
+            self.known.append(line)
+            self._cm.log(f"Warning: core file on {node}: {line}")
```

Comment:
Good point, we don't really need to log which component; `{line}` already tells us the full path for FS cores. For `coredumpctl` cores, it's not really practical to figure out which component anyway.